lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,9 +1,10 @@
-;(function (define) {
+;(function (define, interpolate) {
```

Comment:
It does, but JSHint doesn't understand this. It means that my code gets error underlines on every usage. I was experimenting with whether this was cleaner, because then I only have one error at the bottom of the page.